src/main/java/dansplugins/factionsystem/objects/helper/FactionFlags.java

```diff
@@ -126,26 +127,27 @@ public void sendFlagList(Player player) {
 
     public void setFlag(String flag, String value, Player player) {
         if (flag.equals("neutral") && !configService.getBoolean("allowNeutrality")) {
-            player.sendMessage(ChatColor.RED + "" + localeService.get("NeutralityDisabled"));
+            new PlayerService().sendMessageType(player, ChatColor.RED + "" + localeService.get("NeutralityDisabled")
+                    , "NeutralityDisabled", false);
             return;
         }
 
         if (flag.equals("prefixColor") && !configService.getBoolean("factionsCanSetPrefixColors")) {
-            player.sendMessage("Players can't set prefix colors.");
+            new PlayerService().sendMessageType(player, "&cPlayers can't set prefix colors.", "CannotSetPrefix", false);
             return;
         }
 
         if (flag.equals("prefixColor") && (!configService.getBoolean("playersChatWithPrefixes"))) {
-            player.sendMessage(ChatColor.RED + "" + localeService.get("PrefixesDisabled"));
+            new PlayerService().sendMessageType(player, ChatColor.RED + "" + localeService.get("PrefixesDisabled"), "PrefixesDisabled", false);
             return;
         }
 
         if (flag.equals("fiefsEnabled") && !fiefsIntegrator.isFiefsPresent()) {
-            player.sendMessage("Fiefs either isn't enabled or present.");
+            new PlayerService().sendMessageType(player, "&cFiefs either isn't enabled or present.", "FiefsNotEnable", false);
             return;
         }
 
-        if (flag.equals("officersCanMintCurrency") && !currenciesIntegrator.isCurrenciesPresent()) {
+        if (flag.equals("officersCanMintCurrency") && currenciesIntegrator.isCurrenciesPresent()) {
```

Comment:
This, combined with the currencies integrator change, makes this more difficult to follow. We should be returning if currencies is not present, as the flag isn't supported.